@here/harp-mapview/lib/AnimatedExtrusionHandler.ts

```diff
@@ -0,0 +1,132 @@
+/*
+ * Copyright (C) 2017-2019 HERE Europe B.V.
+ * Licensed under Apache 2.0, see full license in LICENSE
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { TileKey } from "@here/harp-geoutils";
+import { DataSource } from "./DataSource";
+import { MapView } from "./MapView";
+import { Tile } from "./Tile";
+
+/**
+ * Animation states for extrusion effect
+ */
+export enum AnimatedExtrusionState {
+    None,
+    Started,
+    Playing,
+    Finished
+}
+
+/**
+ * Creates `AnimatedExtrusionHandler` that controls animated extrusion effect for
```

Comment:
It doesn't create anything, it 'is'.

More about architecture,

When i looked at this class i envisioned it will handle extrusion in whole MapView or in Tile.

Now, i see this one manages extrusion in MapView, but it's just a simple list.

I don't like that `Tile` which is already huge class gets new almost ten members and methods for this feature, i think that we need separate class also for `AnimatedExtrusionTileHandleer`.

And ofc, describe difference and relationship between those two classes.

Reply:
Refactored. As you suggested, now it has two classes AnimatedExtrusionHandler and AnimatedExtrusionTileHandler